docs/orchestrating-elastic-stack-applications/elasticsearch/bundles-plugins.asciidoc

```diff
@@ -25,11 +25,37 @@ This option has the following pros and cons:
 This option has the following pros and cons:
 +
 * Pro: Easier to get started and upgrade versions.
-* Con: Requires pods to have internet access.
+* Con: Requires pods to have internet access. *See <<istio-note,note>> below about using Istio*
 * Con: Adding new Elasticsearch nodes could randomly fail due to network issues or bad configuration.
 * Con: Each Elasticsearch node needs to repeat the download, wasting bandwidth and slowing startup.
 * Con: Deployment manifests are more complicated.
 
+[id="istio-note"]
+**Note when using Istio**
```

Comment:
This makes it look like in the rendered version as if every below applies to Istio which it does not.

Reply:
Moved Istio note to the end of the doc.  LMK if this is acceptable.